python/openmldb/dbapi/dbapi.py

```diff
@@ -343,14 +343,9 @@ def get_databases(self):
         return self.connection._sdk.getDatabases()
 
     def fetchone(self):
-
-        if self._resultSet is None: return "call fetchone"
+        if self._resultSet is None: raise Exception("query data failed")
```

Comment:
Throw the exception which is defined in DB API. Here may be `DataError` or `DatabaseError`.

Reply:
updated.